content/en/dev/setup.md

```diff
@@ -50,30 +50,37 @@ Run the following commands in the project directory:
 
 ```sh
 bundle install
-yarn install
+yarn
 ```
 
+{{<hint style="info">}}
+If you would like to avoid installing the gems globally, and are not using something like [RVM gemsets](https://rvm.io/gemsets/basics), you may want to configure the bundle path:
+
+```
+bundle config set --local path vendor/bundle
+```
+{{</hint>}}
+
+{{<hint style="info">}}
+On some operating systems, such as Debian and Fedora, `yarn` has been renamed to `yarnpkg`. On these systems, run `yarnpkg` instead of `yarn`.
```

Comment:
Fedora isn't covered by the prerequisites page, and on Debian yarn will come as part of installing Node.js via Nodesource.